dpti/workflows/simulations/base.py

```
@@ -0,0 +1,436 @@
+        #             name=task_name,
+        #             cache_key_fn=task_input_json_hash,
+        #             persist_result=True,
+        #             refresh_cache=REFRESH_CACHE
+        #         )(method)
+        #         setattr(cls, method_name, decorated_method)
+        # cls._class_name = name
+        return cls
+
+
+# %%
+def workflow_task(method_name: str, **task_kwargs):
```

Ruff (0.11.9)

Redefinition of unused `workflow_task` from line 28

(F811)